### PR TITLE
[M775] Fix dashboard org filtering bug

### PR DESCRIPTION
**Pull request by tienhoah**
- Refactor query strings to work correctly with special characters (such as '&' for organization 'Fauna and Flora International').
- Refactor bounding box after filtering sites from Filter Modal.

### CODE REVIEW

**Comment by tienhoah** [comment]
[dashboard org filtering bug?](https://trello.com/c/Sdljc9A3/775-dashboard-org-filtering-bug)

File: src/components/LeafletMap.js [hunk 1]
```diff
@@ -603,7 +603,11 @@ class LeafletMap extends Component {
     if (markersData.length === 0) {
       this.map.setView(mapProperty.center, mapProperty.zoom)
     } else {
-      const mapBounds = markersCluster.getBounds().pad(0.1)
+      const markerBounds = markersCluster.getBounds()
```
Comment (saanobhaai) [inline src/components/LeafletMap.js:606]: Do you want to keep the `.pad(0.1)`?

**Review by saanobhaai — CHANGES_REQUESTED** [review]
I think it looks good, though I'm not entirely clear how it fixes the bugs. Just one question before we merge to dev and try it out.

File: src/components/LeafletMap.js (hunk 1 above)
Comment (saanobhaai) [reply src/components/LeafletMap.js:606]: OK that makes sense

**Pull request merged**